src/base.ts

```diff
@@ -1,14 +1,6 @@
+import { Request } from 'express';
 import { ReadonlyContext } from './context';
 
-export interface Request {
```

Comment:
`express-validator.Request` is a more generic type than `express.Request`. Furthermore in the internal API we don't require all those methods and types defined in `express.Request` that are causing problems with the CI.